shared/data/open-source.json

```
@@ -0,0 +1,142 @@
+[
+  {
+    "name": "Bootstrap",
+    "url": "https://getbootstrap.com/",
+    "version": "5.3.0",
+    "description": "The most popular front-end framework for developing responsive, mobile first projects on the web.",
+    "license": "MIT"
+  },
+  {
+    "name": "IMask",
+    "url": "https://imask.js.org/",
+    "version": "7.6.1",
+    "description": "Vanilla JavaScript input masking library with support for various mask types and dynamic behavior.",
+    "license": "MIT"
+  },
+  {
+    "name": "Autosize",
+    "url": "https://www.jacklmoore.com/autosize/",
+    "version": "6.0.1",
+    "description": "Automatically adjusts textarea height to fit text content.",
+    "license": "MIT"
+  },
+  {
+    "name": "ApexCharts",
+    "url": "https://apexcharts.com/",
+    "version": "3.54.1",
+    "description": "Modern charting library offering interactive and responsive charts.",
+    "license": "MIT"
+  },
+  {
+    "name": "noUiSlider",
+    "url": "https://refreshless.com/nouislider/",
+    "version": "15.8.1",
+    "description": "Lightweight JavaScript range slider with multi-touch support and extensive customization options.",
+    "license": "MIT"
+  },
+  {
+    "name": "CountUp.js",
+    "url": "https://inorganik.github.io/countUp.js/",
+    "version": "2.8.0",
+    "description": "Animates numerical values by counting up to them.",
+    "license": "MIT"
+  },
+  {
+    "name": "List.js",
+    "url": "https://listjs.com/",
+    "version": "2.3.1",
+    "description": "Adds search, sort, and flexibility to tables, lists, and various HTML elements.",
+    "license": "MIT"
+  },
+  {
+    "name": "Masonry",
+    "url": "https://masonry.desandro.com/",
+    "version": "4.2.2",
+    "description": "JavaScript grid layout library that arranges elements optimally based on available vertical space.",
+    "license": "MIT"
+  },
+  {
+    "name": "Mapbox GL JS",
+    "url": "https://docs.mapbox.com/mapbox-gl-js/",
+    "version": "1.8.0",
+    "description": "JavaScript library for interactive, customizable vector maps on the web.",
+    "license": "BSD-3-Clause"
+  },
+  {
+    "name": "Litepicker",
+    "url": "https://litepicker.com/",
```

This leads to an unrelated spam page now. Seems like a reasonable time to choose a replacement, as recommended in https://github.com/tabler/tabler/issues/2242 ?

---

```suggestion
    "url": "https://github.com/wakirin/Litepicker",
```

Probably time to replace Litepicker with something that is actually alive.
I tried so many datepicker libraries, unfortunately none of them came close to Litepicker.